Parallel_test_on_Kobiton_devices_by_using_Katalon/GlobalVariables.glbl

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should organize files as

```
|- java
|- katalon
       |- README.md <-- put the guideline here
       |- ... <-- other files 
```

The README should give:

- One or two statements for introducing Katalon and how to install it (link to its homepage)
- How to register trial account (link to Khanh Do post on blog site)
- How to run this project: update api key, replace device name, etc.

Reply:
ok a